test/unit/multi_response_test.rb

```diff
@@ -172,4 +172,64 @@ def test_handles_ignores_optional_request_result
 
     assert m.success?
   end
+
+  def test_handles_response_with_avs_and_cvv_result
+    r1 = Response.new(true, '1', {}, { avs_result: AVSResult.new(code: 'Y'), cvv_result: 'M' })
+    r2 = Response.new(true, '2', {})
+    m = MultiResponse.run do |r|
+      r.process { r1 }
+      r.process { r2 }
+    end
+    assert_equal [r1, r2], m.responses
+    assert_equal m.avs_result, { 'code' => 'Y', 'message' => 'Street address and 5-digit postal code match.', 'street_match' => 'Y', 'postal_match' => 'Y' }
+    assert_equal m.cvv_result, { 'code' => 'M', 'message' => 'CVV matches' }
+  end
+
+  def test_handles_responses_overwriting_cvv_and_avs_result
```

Comment:
💬 Comment:
The test title should be "..not_overwriting_existing..." because it is taken the latest cvv/avs if available

Reply:
Renamed